source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -1287,6 +1293,13 @@ def make_mock_kirin_item(vj_id, date, status='canceled', new_stop_time_list=[],
     trip.start_date = date
     trip.Extensions[kirin_pb2.contributor] = rt_topic
 
+    if effect == 'modified':
```

Comment:
Minor (to be decided once other comments are decided): it would be better to have enum from the beginning.
But it's a helper and the enum can be long, depending on what changes from other comments, it may not be worth it.